src/sql/src/error.rs

```diff
@@ -56,7 +56,6 @@ pub enum Error {
     UnsupportedDefaultValue {
         column_name: String,
         expr: Expr,
-        backtrace: Backtrace,
```

Comment:
1. since we are removing `Backtrace` field will it make sense to remove `backtrace_opt` method as well?
2. should we do anything with the macro` #[snafu(backtrace)]` as a part of this PR?

Reply:
I think we could leave them unchanged. Since `Error::InvalidDefault` might capture a backtrace.

Reply:
Thanks for the explanations @evenyag